storage_visualization/main.py

```diff
@@ -19,12 +20,16 @@
     remote_tmpdir,
     output_path,
 )
-from cpg_utils.slack import send_message
+from cpg_utils.slack import upload_file
+
+DOCKER_IMAGE = (
+    'australia-southeast1-docker.pkg.dev/cpg-common/images/storage-visualization:latest'
+)
 
 
 def prepare_job(job, clone_repo):
     """Sets up the given job to run scripts in the same repository."""
-    job.image(get_config()['workflow']['driver_image'])
+    job.image(DOCKER_IMAGE)
```

Comment:
Can you only use the `kaleido` image just for the job that needs it - that way we don't have to consistently keep it up to date.

Reply:
See below -- isn't it better to make sure everything is consistently up-to-date?